### PR TITLE
Tidying up

### PR DESCRIPTION
**Pull request by Rumyra**
- Removes draft
- Removes events sub heading (not sure why it was there)
- Moves parse method to static method section
- Adds definitions for toSum and type

### CODE REVIEW

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
I've grabbed this one ;-)

Few comments for your to address. Cheers @Rumyra !

File: files/en-us/web/api/cssnumericvalue/index.html [hunk 1]
```diff
@@ -61,9 +57,14 @@ <h2 id="Methods">Methods</h2>
 	<dt>{{domxref('CSSNumericValue.to')}}</dt>
 	<dd>Converts <code>value</code> into another one with the specified <var>unit.</var></dd>
 	<dt>{{domxref('CSSNumericValue.toSum')}}</dt>
-	<dd>TBD</dd>
+	<dd>Converts the object's value to a {{domxref("CSSMathSum")}} object to values of the specified unit.</dd>
```
[the remark this answers is not in the document]
Comment (Rumyra) [reply files/en-us/web/api/cssnumericvalue/index.html:60]: Haha - this is taken from the member page - I'll update both 👍

File: files/en-us/web/api/cssnumericvalue/index.html [hunk 2]
```diff
@@ -61,9 +57,14 @@ <h2 id="Methods">Methods</h2>
...
 	<dt>{{domxref('CSSNumericValue.type')}}</dt>
-	<dd>TBD</dd>
+	<dd>Returns the type of <code>CSSNumericValue</code>, one of <code>angle</code>, <code>flex</code>, <code>frequency</code>, <code>length</code>, <code>resolution</code>, <code>percent</code>, <code>percentHint</code>, or <code>time</code>.</dd>
+</dl>
+
+<h2 id="Static_Methods">Static Methods</h2>
```
[the remark this answers is not in the document]
Comment (Rumyra) [reply files/en-us/web/api/cssnumericvalue/index.html:65]: So interestingly Joe has pushed back on titles not being title case - unless I've muddled it up, but not looking at recent docs I've done ☺️ I think I'm going to be changing quite a few headings...

File: files/en-us/web/api/cssnumericvalue/index.html [hunk 3]
```diff
@@ -37,10 +37,6 @@ <h2 id="Properties">Properties</h2>
 
 <p>None.</p>
 
-<h3 id="Event_handlers">Event handlers</h3>
-
-<p>No</p>
-
 <h2 id="Methods">Methods</h2>
```
[the remark this answers is not in the document]
Comment (Rumyra) [reply files/en-us/web/api/cssnumericvalue/index.html:40]: So I decided this as I had seen it somewhere else and it seemed like a quick win whilst we're deciding the _correct_ way. I quite like h2 methods and h3 sub headers for instance and static - it means existing pages need not change. But sorry I'm discussing this in the wrong place 😆

**Review by chrisdavidmills — APPROVED** [review]
Looks good for now; thanks @Rumyra !

**Pull request merged**